docs/user/CHANGELOG/index.md

```diff
@@ -48,6 +48,8 @@ Given a version number `X.Y.Z`,
 
     **Changes:**
 
+    - `Input_file` in the `Hydro` and `Afterburner` modules now can be used to modify the expected file names from the appropriate :file_folder: ***IC*** and :file_folder: ***Sampler*** sub-folders, when a string without a "/" character is given.
```

Comment:
You should pick an emoji instead of a bullet point. Used ones are just listed above. I guess `:new:` is appropriate here. I would also use `` `/` `` instead of `"/"`.

```suggestion
    :new: The `Input_file` configuration key in the `Hydro` and `Afterburner` modules can now be used to modify the expected file names from the appropriate :file_folder: ***IC*** and :file_folder: ***Sampler*** sub-folders, when a string without a `/` character is given.
```